### DISCUSSION TITLE
How to enable mixed language during inference of yourtts ?

### DISCUSSION THREAD

**Discussion by mrakotos** [idiap/coqui-ai-TTS#104] [Q&A] [answered]
Hi, 
Does anyone knows how to run inference with mixed language in it. Author of the yourtts paper claim that we can do this but , I try to search for how to do it I didnt find any anwser. By that I mean making inference for the sentence a feeling of "déja-vu" where déja-vu where déja vu is prononced as in french but the rest in english. If anyone knows how to do that please tell us.
Thank, you.

**Comment by eginhard** [comment, marked as answer]
The language embedding in Vits/YourTTS models is concatenated to each character in the text encoder: https://github.com/idiap/coqui-ai-TTS/blob/073f8de652cd4a203aef534e054c0e4580b5deec/TTS/tts/layers/vits/networks.py#L73
You can set a different language embedding for the respective characters to achieve this.
